crates/primitives/src/eip7702/bytecode.rs

```diff
@@ -0,0 +1,84 @@
+use crate::{bytes, Address, Bytes};
+
+/// EIP Version Magic in u16 form.
+pub const EIP7702_MAGIC: u16 = 0xEF01;
+
+/// EOF magic number in array form.
+pub static EIP7702_MAGIC_BYTES: Bytes = bytes!("ef01");
```

Comment:
```suggestion
pub static EIP7702_MAGIC_BYTES: Bytes = bytes!("ef0100");
```

As of the [latest commit](https://github.com/ethereum/EIPs/blob/52de6be37bfb7c134d0a2280f3d5b3d2433dc572/EIPS/eip-7702.md), this should be `0xef0100` ?

Reply:
MAGIC filed is something similar to EOF magic, so I will leave this const as same, and add additional const `EIP7702_VERSION`